.github/workflows/release.yaml

```diff
@@ -0,0 +1,76 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            new_version:
```

Comment:
Can you provide a sample values for parameters like in https://github.com/project-codeflare/instascale/blob/e3df822968e6617cf5e2bcc60469268a42f55738/.github/workflows/instascale-release.yml#L11 ?

Reply:
Done. :+1: